lib/modules/apostrophe-attachments/lib/api.js

```diff
@@ -153,6 +162,18 @@ module.exports = function(self, options) {
         function(callback) {
           self.uploadfs.copyImageIn(tempFile, croppedFile, { crop: crop }, callback);
         },
+        // Question: enable this is if we needed length for cropped images
```

Comment:
This would not be an appropriate way to check as uploadfs is used for the purpose of ensuring it doesn't matter if files are local or in s3, etc. Assuming they live in the filesystem is against the rules (:

We don't use the length property for much (which must be evident since it wasn't working properly yet). It's a bit misleading really since we provide various scaled versions anyway.

I think for now we'll just accept your change for correctly recording the raw original size, which could be useful as something like a rough cut filter for PDFs that are bigger than the user likes... maybe someday (: